orbital-mechanics/TheRealLecture1/SpacecraftDynamicsWeek1r;$.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<p>For some reason the first equation hasn’t worked. Again probably an extra set of characters somewhere or indentation. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angadhn/SpacecraftDynamics/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>